### PR TITLE
Feat/event details

### PR DESCRIPTION
**Pull request by GitAndrew**
Added placeholder Events list
Added placeholder screen for Event Details
Refactored some of the redux implementation

### CODE REVIEW

File: src/redux/store.ts [hunk 1]
```diff
@@ -1,5 +1,8 @@
-import { createStore, applyMiddleware } from "@reduxjs/toolkit";
-import thunk from "redux-thunk";
+import { configureStore } from "@reduxjs/toolkit";
 import rootReducer from "./reducer";
 
-export default createStore(rootReducer, applyMiddleware(thunk));
+const store = configureStore({ reducer: rootReducer });
```
Comment (GitAndrew) [inline src/redux/store.ts:4]: configureStore is redux toolkit's wrapper around createStore which includes thunk
https://redux-toolkit.js.org/usage/usage-guide#simplifying-store-setup-with-configurestore

File: src/screens/Events/EventDetails/EventDetails.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      <View>
+        <Text style={styles.text}>Event Details</Text>
+        {!loading && data.getFoodDriveById && (
+          <>
+            <Text style={styles.text}>Name: {data.getFoodDriveById.name}</Text>
+            <Text style={styles.text}>
+              Description: {data.getFoodDriveById.description}
+            </Text>
+            <Text style={styles.text}>
+              Start Date: {data.getFoodDriveById.startDate}
```
Comment (chrisrabe) [inline src/screens/Events/EventDetails/EventDetails.tsx:41]: I think we have to convert this from UTC to locale date - use dayjs

File: src/screens/Events/EventDetails/EventDetails.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            </Text>
+            <Text style={styles.text}>
+              Address: {data.getFoodDriveById.location.address}
+            </Text>
+            <Text style={styles.text}>
+              Contact Number: {data.getFoodDriveById.contactNumber}
+            </Text>
+            <Text style={styles.text}>
+              Allergens:{" "}
+              {data.getFoodDriveById.food
```
Comment (chrisrabe) [inline src/screens/Events/EventDetails/EventDetails.tsx:54]: Do we want to use tags for this?

**Review by chrisrabe — CHANGES_REQUESTED** [review]
- use tags component to display allergens
- need to convert UTC date to locale time using dayjs

File: src/screens/Events/EventDetails/EventDetails.tsx (hunk 3 above)
Comment (GitAndrew) [reply src/screens/Events/EventDetails/EventDetails.tsx:54]: Yeah this code/screen is placeholder for now, will update it to look like the designs in another PR

File: src/screens/Events/index.tsx [hunk 4]
```diff
@@ -1,11 +1 @@
-import React from "react";
```
Comment (winniezheng) [inline src/screens/Events/index.tsx:1]: index.ts instead of tsx?

**Review by chrisrabe — APPROVED** [review]

**Pull request merged**